torchgeo/datasets/fair1m.py

```diff
@@ -175,9 +175,9 @@ class FAIR1M(NonGeoDataset):
             os.path.join('validation', 'images'),
             os.path.join('validation', 'labelXml'),
         ),
-        'test': (os.path.join('test', 'images')),
+        'test': (os.path.join('test', 'images'),),
```

Comment:
This was an actual bug in the dataset uncovered by mypy once I added type hints!